rkt/plan.sh

```diff
@@ -0,0 +1,29 @@
+pkg_name=rkt
+pkg_description="rkt - the pod native container engine"
+pkg_origin=core
+pkg_version=1.29.0
+pkg_maintainer="The Habitat Maintainers <humans@habitat.sh>"
+pkg_license=('Apache-2')
+pkg_source=https://github.com/rkt/rkt/releases/download/v${pkg_version}/rkt-v${pkg_version}.tar.gz
+pkg_upstream_url=https://github.com/rkt/rkt/releases
+pkg_shasum=4fdd3d46aa58a9e7a7d1c36a7cb390ea96fd297b694295eb79b8c185dbc73e7e
+pkg_dirname=rkt-v${pkg_version}
+pkg_deps=(core/gnupg core/glibc)
```

Comment:
Not blocking, just curious: Do you know where gnupg is used?  I don't see it linked in.  Does it get dlopen'd or something?

Reply:
i think it actually reaches out to gpg binary to sign its containers, so i think this is just ensuring it's on disk, now WRT whether it will be found by the rkt binary i don't know yet. This is a carry of a very old PR :)